### PR TITLE
Rust/60x/v1

### PR DESCRIPTION
**Pull request by victorjulien**
See if we can encode MSRV in a file in the dist, and then use it.

### CODE REVIEW

File: rust/Cargo.toml.in [hunk 1]
```diff
@@ -2,6 +2,7 @@
 name = "suricata"
 version = "@PACKAGE_VERSION@"
 edition = "2018"
+rust-version = "1.41.1"
```
Comment (inashivb) [inline rust/Cargo.toml.in:5]: In https://doc.rust-lang.org/nightly/cargo/reference/manifest.html#the-rust-version-field, I read `The first version of Cargo that supports this field was released with Rust 1.56.0. In older releases, the field will be ignored, and Cargo will display a warning.`
Q: Is it correct to try and use it to check for older versions?

Comment (victorjulien) [reply rust/Cargo.toml.in:5]: yeah the warning is expected for 1.41, but harmless

Comment (victorjulien) [reply rust/Cargo.toml.in:5]: the field is used here https://github.com/OISF/suricata/pull/10674/files#diff-a76cf7978f0a981f911e8d68d2351a72a268977304612226433df4fb8203b06fR1231

File: .github/workflows/builds.yml [hunk 2]
```diff
@@ -1221,15 +1221,15 @@ jobs:
                 python3-yaml \
                 zlib1g \
                 zlib1g-dev
-      - name: Install Rust
-        run: curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain ${RUST_VERSION_MIN} -y
```
Comment (catenacyber) [inline .github/workflows/builds.yml:1225]: `RUST_VERSION_MIN` is unused and its definition can go away (as in 7) right ?

Comment (jasonish) [reply .github/workflows/builds.yml:1225]: Related note for later.. I know don't RUST_VERSION_KNOWN provides any value.  Its use can probably be replaced with latest stable.  We already have some that pull rustc from packages, that's a better indicator.

**Review by jasonish — APPROVED** [review]

**Pull request merged**